wpilibNewCommands/src/main/native/cpp/frc2/command/RamseteCommand.cpp

```diff
@@ -146,6 +146,10 @@ void RamseteCommand::Execute() {
 
 void RamseteCommand::End(bool interrupted) {
   m_timer.Stop();
+
+  if (interrupted) {
+    m_outputVel(0.0, 0.0);
```

Comment:
You need to choose the correct consumer based on whether PID is being calculated in the command or not.